app/models/workarea/pricing/collection.decorator

```diff
@@ -1,81 +1,111 @@
 module Workarea
   decorate Pricing::Collection, with: :flow_io do
-    decorated { attr_accessor :flow_experience }
-
-    def regular_min_price
-      return super unless flow_experience.present?
-
-      @regular_min_price ||= records
-        .map { |pricing_sku| pricing_sku.find_price(experience: flow_experience).regular }
-        .sort
+    def regular_min_price_sku
+      @regular_min_price_sku ||= generic_prices
+        .map { |p| [p.sku, p.regular] }
+        .sort_by { |(_sku, regular)| regular }
         .first
+        &.first
```

Comment:
If you've got an array here to `map` and `sort_by`, it won't be `nil` when you call `first`.

Reply:
not sure I understand...